### PR TITLE
#2838 invite dialog with meeting context

### PR DESCRIPTION
**Pull request by ackernaut**
Addresses #2838
### Tests
If the team has an active meeting when the invitation is viewed, the invitation dialog has the abstract meeting background and contextual copy to join meeting:
  - [ ] New user, sign up using Google
  - [ ] New user, sign up using email/password
  - [ ] Team invitation already accepted (shows link to join active meeting)
  - [ ] Existing user sign in using Google
  - [ ] Existing user sign in using email/password

If the team doesn’t have an active meeting when the invitation is viewed, the invitation dialog has the basic background and default copy to join team:
  - [ ] New user, sign up using Google
  - [ ] New user, sign up using email/password
  - [ ] Team invitation already accepted (shows link to join active meeting)
  - [ ] Existing user sign in using Google
  - [ ] Existing user sign in using email/password

Invite dialogs have generic background at all times:
- [ ] Invitation not found
- [ ] Invitation expired

### CODE REVIEW

File: src/universal/components/TeamInvitationGoogleCreateAccount.tsx [hunk 1]
```diff
@@ -140,6 +144,7 @@ export default createFragmentContainer(
   withAtmosphere(withMutationProps(withRouter(TeamInvitationGoogleCreateAccount))),
   graphql`
     fragment TeamInvitationGoogleCreateAccount_verifiedInvitation on VerifiedInvitationPayload {
+      meetingType
```
Comment (ackernaut) [inline src/universal/components/TeamInvitationGoogleCreateAccount.tsx]: @mattkrick I’m stumped as to why `meetingType` doesn’t come through in the payload. This is the only hint:
```
index.js:3740 Warning: RelayResponseNormalizer(): Payload did not contain a value for field `meetingType: meetingType`. Check that you are parsing with the same query that was used to fetch the payload.
```

Comment (mattkrick) [reply src/universal/components/TeamInvitationGoogleCreateAccount.tsx]: likely a stale fragment on the server. try restarting the server, if that doesn't work, maybe running `yarn postdeploy`?

**Comment by ackernaut** [comment]
Ready for review @mattkrick

File: src/universal/components/TeamInvitationEmailCreateAccount.tsx [hunk 2]
```diff
@@ -26,17 +27,26 @@ const TeamName = styled('span')({
...
+      <DialogTitle>
+        {meetingType ? `Join ${meetingTypeToLabel[meetingType]} Meeting in Progress` : 'Join Team'}
+      </DialogTitle>
       <DialogContent>
-        <InvitationDialogCopy>
-          Choose a password for immediate access to your team: <TeamName>{teamName}</TeamName>
-        </InvitationDialogCopy>
+        {meetingType ? (
+          <InvitationDialogCopy>
+            You’re just 1 step away from participating with your team:{' '}
```
Comment (mattkrick) [inline src/universal/components/TeamInvitationEmailCreateAccount.tsx]: This says 1 step, but it doesn't say what that 1 step is. I prefer "Choose a password for immediate access" here
You can also reduce the size of the conditional:

```
<InvitationDialogCopy>
{meetingType ? meetingCopy : teamCopy}
<TeamName>{teamName}</TeamName>
 </InvitationDialogCopy>
```

File: src/universal/components/TeamInvitationEmailSignin.tsx [hunk 3]
```diff
@@ -35,9 +36,16 @@ const TeamInvitationEmailSignin = (props: Props) => {
       <Helmet title={`Sign in | Team Invitation`} />
       <DialogTitle>Welcome back, {preferredName}!</DialogTitle>
       <DialogContent>
-        <InvitationDialogCopy>
-          Enter your password for immediate access to your team: <TeamName>{teamName}</TeamName>
-        </InvitationDialogCopy>
+        {meetingType ? (
+          <InvitationDialogCopy>
+            Enter your password to join the {meetingTypeToLabel[meetingType]} Meeting for: (
```
Comment (mattkrick) [inline src/universal/components/TeamInvitationEmailSignin.tsx]: nit: start conditional here to make it a little smaller

File: src/universal/components/TeamInvitationGoogleCreateAccount.tsx [hunk 4]
```diff
@@ -99,21 +100,32 @@ class TeamInvitationGoogleCreateAccount extends Component<Props, State> {
...
           <CopyMargins>
             <InvitationDialogCopy>
               It looks like your email is hosted by Google.
             </InvitationDialogCopy>
-            <InvitationDialogCopy>
-              Tap below for immediate access to your team: <TeamName>{teamName}</TeamName>
-            </InvitationDialogCopy>
+            {meetingType ? (
+              <InvitationDialogCopy>
+                You’re just 1 step away from participating with your team:{' '}
```
Comment (mattkrick) [inline src/universal/components/TeamInvitationGoogleCreateAccount.tsx]: same as above, I'm not sure what "1 step" means. "tap below for immediate access" seems clearer to me

can also start conditional here

File: src/universal/components/TeamInvitationEmailSignin.tsx (hunk 3 above)
Comment (mattkrick) [reply src/universal/components/TeamInvitationEmailSignin.tsx]: Remove parens around team name `Meeting for: (*Parabol Testing)`

File: src/universal/components/TeamInvitationGoogleCreateAccount.tsx [hunk 5]
```diff
@@ -99,21 +100,32 @@ class TeamInvitationGoogleCreateAccount extends Component<Props, State> {
...
+    const {meetingType, teamInvitation, teamName} = verifiedInvitation
     if (!teamInvitation) return null
     const {email} = teamInvitation
     return (
       <StyledDialog>
         <Helmet title={`Sign up with Google | Team Invitation`} />
-        <DialogTitle>Welcome!</DialogTitle>
+        <DialogTitle>
+          {meetingType
+            ? `Join ${meetingTypeToLabel[meetingType]} Meeting in Progress`
```
Comment (mattkrick) [inline src/universal/components/TeamInvitationGoogleCreateAccount.tsx]: title is too long, consider `Join Meeting`

Screenshot from 2019-06-18 07-34-39

**Pull request merged**